### PR TITLE
[ITG-79] Setup CI

### PR DESCRIPTION
**Pull request by john-sonz**
CI setup for `bode` and `cabra`.

---

#### Resources

https://kulawska.atlassian.net/browse/ITG-79

### CODE REVIEW

File: .github/workflows/cabra_ci.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: "Cabra CI"
+
+on:
+  pull_request:
+    paths:
+      - "cabra/**"
+    branches: [develop, master]
```
Comment (tgargula) [inline .github/workflows/cabra_ci.yaml:7]: I would add it here as well.
```
  push:
    branches: [develop, master]
```

File: .github/workflows/bode_ci.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+name: "Bode CI"
+
+on:
+  pull_request:
+    paths:
+      - "bode/**"
+    branches: [develop, master]
```
[the remark this answers is not in the document]
Comment (john-sonz) [reply .github/workflows/bode_ci.yaml:7]: Well, why not 💸 💸 💸 
I think we can go even further and do
```
on:
    pull_request:
        paths:
          - "cabra/**"
     push:
        paths:
          - "cabra/**"
```

File: .github/workflows/bode_ci.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
+name: "Bode CI"
+
+on:
+  pull_request:
+    paths:
+      - "bode/**"
+  push:
+    paths:
+      - "bode/**"
```
Comment (tgargula) [inline .github/workflows/bode_ci.yaml:9]: I think every push means also running after pushing from local -- it may be doubled and unnecessary. Wdyt?

Comment (john-sonz) [reply .github/workflows/bode_ci.yaml:9]: Yeah, this way you can see if CI is passing without creating a PR, the feedback loop is a bit shorter. I think we can leave it like that, merge, observe if we have redundant workflow runs and then adjust accordingly if needed

Comment (tgargula) [reply .github/workflows/bode_ci.yaml:9]: ok

**Review by tgargula — APPROVED** [review]
<3

**Pull request merged**